lisa/target.py

```diff
@@ -728,6 +730,30 @@ def remove_kmodule(self, kmodule):
 
         self.target.execute('rmmod {}'.format(kmodule))
 
+    def _init_extend_sched_events(self):
+        """
+        On newer kernels (5.3 and above) PELT signals are present as bare
+        tracepoints and we need to load a module to convert them to proper
+        TRACE_EVENTs.
+
+        This function automatically handles this.
+        """
+
+        sched_events_path = '/sys/kernel/debug/tracing/events/sched/'
+
+        sched_load_avg_task = os.path.join(sched_events_path, 'sched_load_avg_task')
```

Comment:
Since you use `os.path.join` here, shouldn't we do the same for `sched_events_path`?
Moreover, we should use devlib's `target.os.path.join` since these are target specific paths.

Reply:
Moreover, the base should be probably collected from the target via `target.list_file_systems()` and looking for an `FstabEntry` with `fs_type='tracefs'`

Reply:
Something along these lines should make it:
```python
for entry in target.list_file_systems():
    if entry.fs_type != 'tracefs':
        continue
target.path.join(entry.mount_point, 'events', 'sched')
```

Reply:
The code was using mixed stuff to be honest and I got lost in that. I'll fix it.